packages/mask/src/site-adaptors/mirror.xyz/injection/Tips/MenuAuthorTipButton.tsx

```diff
@@ -1,85 +1,27 @@
-import { useMemo } from 'react'
 import { MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
-import {
-    createInjectHooksRenderer,
-    Plugin,
-    useActivatedPluginsSiteAdaptor,
-    useIsMinimalMode,
-} from '@masknet/plugin-infra/content-script'
-import { EMPTY_LIST, PluginID, type SocialAccount } from '@masknet/shared-base'
-import { makeStyles } from '@masknet/theme'
-import type { Web3Helper } from '@masknet/web3-helpers'
-import { useWeb3Others, useNetworkContext, DefaultWeb3ContextProvider } from '@masknet/web3-hooks-base'
-import { useCurrentVisitingIdentity } from '../../../../components/DataSource/useActivatedUI.js'
+import { Plugin } from '@masknet/plugin-infra'
+import { DefaultWeb3ContextProvider } from '@masknet/web3-hooks-base'
 import { attachReactTreeWithContainer } from '../../../../utils/shadow-root/renderInShadowRoot.js'
-import { menuAuthorSelector as selector } from '../../utils/selectors.js'
 import { startWatch } from '../../../../utils/startWatch.js'
+import { querySelector } from '../../utils/selectors.js'
+import { TipsButtonWrapper } from './TipsButtonWrapper.js'
+
+function selector() {
+    return querySelector(
+        [
+            'div[style$="height: 56px;"] a',
```

Comment:
Matching a style attribute is easy to break.

Reply:
Then will fallback to `div:has(> div > button[data-state="closed"]) a` which is more relaible.

Reply:
Move `div:has(> div > button[data-state="closed"]) a` to the first.